### DISCUSSION TITLE
Daily entries on the weekly page

### DISCUSSION THREAD

**Discussion by inzagi1** [stdword/logseq13-full-house-plugin#42] [Showroom]
Hi,
Can you please advise me, because I can't find an answer to my question anywhere.
With the plugins “full house template” and “show weekday and week number” I want to realize a weekly overview.
I make daily logs for example
`log:: Today was a good day #dayreview.`
And these daily entries for the week, I want to collect on a weekly page, which is created by the plugin “show weekday and week number” as a weekly review.
Can you please tell me if this can be realized with the “full house template” plugin ? maybe a javascript query?

**Comment by stdword** [comment]
Hi, @inzagi1! Check [this](https://github.com/stdword/logseq13-full-house-plugin/discussions/43) out 🙌
